src/components/ModalWindow.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import Modal from "react-modal";
+import { ReviewForm } from "./ReviewForm";
+
+export function ModalWindow(props) {
```

Comment:
この時点で props ばらしてもいいです。
`props` を `props` として使う理由が無ければ基本ばらしておいた方が、何に依存しているか宣言部分でわかるというメリット（？）があります

```suggestion
export function ModalWindow({ modalOpenFlag, closeModal, editableReview, updateList }) {
```